contact.html

```diff
@@ -0,0 +1,159 @@
+    <title>Responsive grid project-contact</title>
+</head>
+
+<body>
+
+     <!-- HEADER -->
+     <section class=" container-fluid fixed-top ">
+ 
+        <div class="row mt-3 mx-5">
+
+            <div class="col-sm-6 col-md-12 d-flex align-items-end flex-column"> 
+               
+                <button type="submit" class="btn btn-primary btn-md"> Book an Appointment</button>     
+                </div>
+                <div class="col-6 col-md-12  d-flex align-items-end flex-column"> 
+
+                <nav class="navbar navbar-expand-lg navbar-light ">
+            
+                    <button class="navbar-toggler " data-toggle="collapse" data-target="#navbarMenu">
+                    <span class="navbar-toggler-icon  "></span>
+                </button> 
+
+                        <div class="collapse navbar-collapse" id="navbarMenu">
+                        
+                        <ul class="navbar-nav">
+                            <li class="nav-item">
+                                <a href="./index.html" class="nav-link">Home</a>
+                            </li>
+                            <li class="nav-item">
+                                <a href="#" class="nav-link">About</a>
+                            </li>
+                            <li class="nav-item">
+                                <a href="./services.html" class="nav-link">Services</a>
+                            </li>
+                            <li class="nav-item">
+                                <a href="#" class="nav-link">Blog</a>
+                            </li>
+                            <li class="nav-item">
+                                <a href="./contact.html" class="nav-link">Contact</a>
+                            </li>
+        
+                        </ul>
+                    </div>              
+                </nav>    
+    </div>
+
+        </div>
+
+ <!-- Maria Hope -->
+        
+        <div class="row">
+
+            <div class=" col-6 offset-3 col-12 offset-md-1 "> 
+            <a class="navbar-brand title fst-italic fs-10 " href="#">Maria Hope, Ph.D. -Psychologist</a>
+        </div>
+         
+        </div>
+
+    </section>
+
+
+
+       <main>
+          <section class="container-fluid contact-back">
+              <div class="row mt-5 mx-5 gx-0 ">
+                <div class="col-12 mt-0 col-6 mt-5 ">
+                    <p class="one mt-5 mb-5 ">Contact Me</p>
+                    <h2 class="two">I'm a title. ​Click here to edit me.</h2>
+                    <p class="three ">I'm a paragraph. Click here to add your own text and edit me. I’m a great place for you to tell a story and let your users know a little more about you.</p>
+                  
+                </div>
+                </div>
+                <div class="row justify-content-center">
+                    <div class="col-md-3  mb-2 ml-3">
+                        <input type="text" class="form-control" placeholder="Name" id="name">
```

Comment:
Your form looks good but it doesn't follow the design. The fields should have a black border on hover and on focus.